### PR TITLE
Update user and Profile

### PR DESCRIPTION
**Pull request by OliverLundhAndersson**
Updates User with specialism and Profile with the fields required for the profile page

### CODE REVIEW

File: prisma/seed.js
Comment (sebgro98) [inline prisma/seed.js:1]: Have to fix prettier in this file

**Review by sebgro98 — CHANGES_REQUESTED** [review]
You also have to provide with the migration file so we can update the table in the database with the new fields

**Review by sebgro98 — CHANGES_REQUESTED** [review]
When testing the endpoint localhost:4000/cohorts/addUser
im getting that this endpoint is giving errors regarding the field 

Unknown field `githubUrl` for select statement on model Profile.

can you please take a look at this

**Review by sebgro98 — DISMISSED** [review]
Approved

**Review by sebgro98 — APPROVED** [review]
Approved

**Review by LudwigJL — APPROVED** [review]
Looks good to me

**Comment by amos1969** [comment]
Approved

**Pull request merged**